.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     container:
       image: rostooling/setup-ros-docker:ubuntu-jammy-latest
     steps:
-    - uses: ros-tooling/action-ros-ci@v0.3
+    - uses: ros-tooling/action-ros-ci@v0.4
```

Comment:
Would you mind also updating the image above to `ghcr.io/ros-tooling/setup-ros-docker/setup-ros-docker-ubuntu-noble`?